opr/functions.py

```diff
@@ -36,3 +38,17 @@ def basic_melting_temperature_calc(sequence):
     else:
         melting_temperature = 64.9 + 41 * ((g_count + c_count - 16.4) / (a_count + t_count + g_count + c_count))
     return melting_temperature
+
+
+def gc_clamp_calc(sequence):
+    """
+    Calculate GC clamp.
+
+    :param sequence: primer nucleotides sequence
+    :type sequence: str
+    :return: Number of guanine (G) or cytosine (C) bases in the last 5 bases of a primer.
```

Comment:
:return: <s>N</s>number of guanine (G) or cytosine (C) bases in the last 5 bases of a primer